modules/standard/Math.chpl

```diff
@@ -155,6 +155,12 @@ Bessel Functions
 :proc:`~Math.y1`
 :proc:`~Math.yn`
 
+.. _math-other:
+
+Remaining Functions
```

Comment:
This is definitely fine as is, but I wouldn't be opposed to calling it "Optimizations" or something along those lines.

Reply:
What do you think about calling it "Builtins"?

Reply:
To me, that would imply that it should be defined in AutoMath.  Which we may still do eventually, but I don't think is the right call while it is still prototypical